assets/src/js/frontend/form-theme/iframe-content.js

```diff
@@ -0,0 +1,20 @@
+const iFrameResizer = {
+	targetOrigin: location.origin,
+
+	onReady: function() {
+		window.parentIFrame.sendMessage( 'giveEmbedFormContentLoaded' );
+	},
+
+	onMessage: function( message ) {
+		console.log( message );
```

Comment:
Careful not to leave `console.log`. 😉

Reply:
@JasonTheAdams I removed it but that must be cleanup ( or suggestion ) by `eslint`. I will look linting tool when I will have time.